src/main/java/racing/controller/RacingGame.java

```diff
@@ -1,64 +1,48 @@
 package racing.controller;
 
-import racing.model.Car;
-import racing.model.CarLineUp;
-import racing.model.RandomNo;
+import racing.model.*;
 import racing.view.InputView;
 import racing.view.OutputView;
 
 public class RacingGame {
-
     private CarLineUp lineUp;
-    private int trialTime;
+    private TrialTime trialTime;
 
-    public void play(){
+    public void play() {
         initializeLineUp();
         initializeTrialTime();
         race();
         showWinners();
     }
 
-    private void initializeLineUp(){
+    private void initializeLineUp() {
         try {
-            lineUp = new CarLineUp();
-            enrollCarToLineUp();
-        } catch (NullPointerException | IllegalArgumentException e){
+            this.lineUp = new CarLineUp(InputView.inputCarNames());
+        } catch (NullPointerException | IllegalArgumentException e) {
             OutputView.printExceptionMessage(e);
             initializeLineUp();
         }
     }
 
-    private void enrollCarToLineUp() {
-        for (String name : InputView.inputCarNames()){
-            lineUp.add(new Car(name));
-        }
-    }
-
-    private void initializeTrialTime(){
+    private void initializeTrialTime() {
         try {
-            trialTime = InputView.inputTrialTime();
-        } catch(Exception e){
+            this.trialTime = new TrialTime(InputView.inputTrialTime());
+        } catch (NullPointerException | IllegalArgumentException e) {
             OutputView.printExceptionMessage(e);
-            initializeLineUp();
+            initializeTrialTime();
         }
     }
 
-    private void race(){
+    private void race() {
         OutputView.printResultMessage();
-        for (int index = 0; index < trialTime; index++){
-            raceOneTime();
+        for (int index = 0; index < trialTime.getTrialTime(); index++) {
```

Comment:
for문이 반복될 때마다 NumberGenerator 객체를 생성할 필요는 없겠네요
```suggestion
        NumberGenerator numberGenerator = new RandomNoGenerator();
        for (int index = 0; index < trialTime.getTrialTime(); index++) {
            lineUp.raceOneTime(numberGenerator);
            OutputView.printOneRoundResult(lineUp);
        }
```